ps_searchbar.php

```diff
@@ -36,6 +36,11 @@
 
 class Ps_Searchbar extends Module implements WidgetInterface
 {
+    /**
+     * @var string Name of the module running on PS 1.6.x. Used for data migration.
+     */
+    private const PS_16_EQUIVALENT_MODULE = 'blocksearch';
```

Comment:
```suggestion
    const PS_16_EQUIVALENT_MODULE = 'blocksearch';
```

Oh shit, it's 5.6 compatible. Sorry